docs/source/guides/using_compose_with_featuretools.ipynb

```diff
@@ -563,7 +563,7 @@
     "Describe Labels\n",
     "---------------\n",
     "\n",
-    "After transforming the labels, we could use :meth:`LabelTimes.describe` to print out the distribution with settings and transforms used to make the labels. This is useful as a reference for understanding how the labels were generated from raw data. We could also see the balance in the labels from the distribution."
+    "After transforming the labels, we could use :meth:`LabelTimes.describe` to print out the distribution with the settings and transforms that were used to make the labels. This is useful as a reference for understanding how the labels were generated from raw data. Also, the label distribution is helpful for determining if we have disproportionate labels."
```

Comment:
maybe imbalanced is better than "disproportionate"?

Reply:
yes, made the update